modifications/update-jenkins-label

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+halp() {
+  cat <<-EOHELP
+	$(basename $0): Script to modify Jenkins image installer files to use the current
+	jenkins rpm.
+	
+	This script expects environment variables \$BREW_EVENT and \$BREW_TAG to be set, and will fail if not.
+EOHELP
+}
+
+get_jenkins_version() {
+  local build version
+  version="$(brew call --json-output listTagged "$BREW_TAG" event="$BREW_EVENT" latest=true package=jenkins | jq -r '.[].version')"
```

Comment:
This actually doesn't 100% compatible with assemblies, because when building an image for an assembly with a basis Brew event, the latest rpm is not used unless it is [pinned by "is"](https://github.com/openshift/ocp-build-data/blob/f78b55f9151c2a0e925ddb1e1932e5eb8b227ed6/releases.yml#L14).

But I am ok to leave your implementation for now. Maybe you can add a `FIXME` comment to highlight the caveat.

Reply:
Pushed a FIXME comment.
Is the plashet repo url known at `images:rebase` time? Could query that instead.